AUTHORS

```diff
@@ -23,6 +23,7 @@ Dmitry Kovega
 Emeric Fermas
 Emmanuel Rodriguez
 Eric Myhre
+Erik Aigner
```

Comment:
Oh, been a long time since this file actually got amended. I'm still not part of it myself :P